Assets/Scripts/Game/UI/DialogueSystem.cs

```diff
@@ -75,6 +78,24 @@ private IEnumerator ShowDialogue(int node)
                 GameManager.QuestSystem.AddNote(note);
             }
 
+            // Apply any dialogue index overrides
+            foreach (DialogueIndexOverride idxOverride in graphNode.dialogueIndexOverrides)
+            {
+                QuestGiver target = currentTarget;
+                if (idxOverride.targetUniqueID != "")
+                {
+                    target = Entity.Find(idxOverride.targetUniqueID).GetComponent<QuestGiver>();
```

Comment:
This is definitely *not* necessary, but it might be cool to have UniqueID wrappers for all the classes that an entity might have. Then we could do `QuestGiver.Find(UID)` too.

Reply:
I was thinking the same thing. But maybe rather than needing a wrapper for each subclass, we just build a generic method:
```cs
// Not tested
public static T Find<T>(string id) where T : Entity
{
    return UniqueID.Get(id).GetComponent<T>();
} 
```

Reply:
👀 

bet